### PR TITLE
feat: support another .eaf format for mic ID identification

### PR DESCRIPTION
**Pull request by eddieantonio**
Support extracting mic ID from `2017-05-18pm-US-Track_04`

### CODE REVIEW

File: validation/models.py [hunk 1]
```diff
@@ -384,9 +384,7 @@ class Recording(models.Model):
     )
 
     comment = models.CharField(
-        help_text="The comment provided in the ELAN file",
-        max_length=256,
-        blank=True,
+        help_text="The comment provided in the ELAN file", max_length=256, blank=True,
```
Comment (eddieantonio) [inline validation/models.py]: uh... black did this for me idk

File: librecval/extract_phrases.py [hunk 2]
```diff
@@ -516,23 +516,26 @@ def get_mic_id(name: str) -> int:
...
             (?:             # An optional date/time code
                 \d{4}       # year
                 -
                 \d{2}       # month
                 -
                 \d{2}       # day
-                (?:[ap]m)
+                (?:[ap]m)   # AM/PM
+                (?:
+                   -[UD]S   # upstairs/downstairs
```
Comment (nienna73) [inline librecval/extract_phrases.py]: Also `OFF` and `KCH`

**Review by nienna73 — CHANGES_REQUESTED** [review]
`get_mic_id` also needs to look for location codes `OFF` and `KCH`

**Review by nienna73 — CHANGES_REQUESTED** [review]
It's still upset with this format: `2017-04-20am-US_Recorded_Track1_001`
It might be the lack of '_', '  ', or '0' after "Track".

This issue isn't pressing, as in it doesn't affect production, so it can wait over the weekend.

**Comment by eddieantonio** [comment]
> It's still upset with this format: `2017-04-20am-US_Recorded_Track1_001`
> It might be the lack of '_', ' ', or '0' after "Track".
> 
> This issue isn't pressing, as in it doesn't affect production, so it can wait over the weekend.

Um... just for posterity's sake: which "1" is the mic number? Is it the `Track1` or the `_001`?

**Comment by nienna73** [comment]
It's the `1` in `Track1`, I confirmed with the filesystem.

**Review by nienna73 — APPROVED** [review]
I think we finally got them all!

**Pull request merged**